File: src/main/java/com/example/ssauc/user/order/dto/OrderRequestDto.java
package com.example.ssauc.user.order.dto;
import lombok.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
@ToString
public class OrderRequestDto {
    private Long productId;
    private Long buyerId;
    private Long sellerId;
    private int totalPayment;
    private String postalCode;
    private String deliveryAddress;
    private String selectedOption;
}

File: src/main/java/com/example/ssauc/user/order/dto/OrderRequestDto.kt
package com.example.ssauc.user.order.dto

import lombok.*

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
@ToString
class OrderRequestDto {
    private val productId: Long? = null
    private val buyerId: Long? = null
    private val sellerId: Long? = null
    private val totalPayment = 0
    private val postalCode: String? = null
    private val deliveryAddress: String? = null
    private val selectedOption: String? = null
}
